### PR TITLE
Fix merge conflicts

### PR DESCRIPTION
**Pull request by cibelius**
* feat: replace QA_ENV with TWO_FA_ENABLED and remove demo scope

* fix: set TWO_FA_ENABLED default to true for production safety

* fix: correct reindex.sh path in Dockerfile.assets

* fix: only set TWO_FA_ENABLED on services that use it

* fix: set DEFAULT_USER_PASSWORD for user-mgnt in dev, QA and e2e environments

* feat: add OPENCRVS_ENVIRONMENT for explicit environment detection in employee seeder

* fix(deploy): add events service TWO_FA_ENABLED=false to non-production compose files## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/data-seeding/employees/handler.ts [hunk 1]
```diff
@@ -8,12 +8,15 @@
...
+import { OPENCRVS_ENVIRONMENT } from '@countryconfig/constants'
 import { readCSVToJSON } from '@countryconfig/utils'
 import { Request, ResponseToolkit } from '@hapi/hapi'
 
 export async function usersHandler(_: Request, h: ResponseToolkit) {
   const users: unknown[] = await readCSVToJSON(
-    './src/data-seeding/employees/source/default-employees.csv'
+    ['staging', 'production'].includes(OPENCRVS_ENVIRONMENT)
+      ? './src/data-seeding/employees/source/prod-employees.csv'
+      : './src/data-seeding/employees/source/default-employees.csv'
```
Comment (Nil20) [inline src/data-seeding/employees/handler.ts:19]: For Farajaland, I think seeding from the `default-employees.csv` is the expected behavior.

File: src/environment.ts [hunk 2]
```diff
@@ -32,6 +32,8 @@ export const env = cleanEnv(process.env, {
   MOSIP_API_USERINFO_URL: url({
     devDefault: 'http://localhost:2024/esignet/get-oidp-user-info'
   }),
+  TWO_FA_ENABLED: bool({ default: true }),
```
Comment (Nil20) [inline src/environment.ts:35]: Since `QA_ENV` was only used for seeding from a specific csv and we do not need it for Farajaland, it shouldn't be necessary
